### ISSUE TITLE
Suggestion: implement status recommendation

### ISSUE THREAD

**Issue by sirkitree** [backdrop-ops/contrib#25]
It might be nice to have a recommendation in the README example that when creating a README the author puts a section for status. Especially in these early days when people won't know if an error they're getting is due to core or a contrib module, it would be better to identify if a module has some sort of status - I'm thinking just a quick note like Status: Stable, or Status: Not yet fully ported, or something along those lines. Anything to indicate that there is an actual working version.

`version` - `status`  perhaps?

Or even to mimic the Drupal.org project pages:

---
Recommended Release/s
* tag 

Development Release/s
* branch

**Comment by quicksketch** [comment]
Long-term we're working on keeping info like this on backdropcms.org itself with the contrib listing there (not yet implemented). But short term it might not be a bad idea to put something in the readme.

However, isn't a release and tag the same thing as a "Recommended release"? I think we should just encourage real releases and consider them to be the stable versions. If they're not ready, they should be marked pre-release, as we did for the Backdrop project itself: https://github.com/backdrop/backdrop/releases

I don't have any projects with prereleases, but I did make a final for Devel (https://github.com/backdrop-contrib/devel/releases) and Extlink (https://github.com/backdrop-contrib/extlink/releases).
